src/components/SliderCarousel/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Carousel } from "react-responsive-carousel";
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+// import { photos } from '../../constants';
+import { slides } from '../../constants';
+
+export const SliderCarousel = () => (
+    <div className="slider-container">
+      <div className="moto-parent">
+        <p className="moto">Show your natural<br/> beauty to the world</p>
+      </div>
+      <Carousel showThumbs={false} showStatus={false} dynamicHeight={true} 
```

Comment:
Replace dynamicHeight={true} with dynamicHeight

https://ru.reactjs.org/docs/jsx-in-depth.html#props-default-to-true